### PR TITLE
[Subrip] - Keep text unchanged when loading.

### PR DESCRIPTION
**Pull request by ivandrofly**
Load subtitles keeping the trailing

### CODE REVIEW

File: libse/SubtitleFormats/SubRip.cs [hunk 1]
```diff
@@ -83,8 +83,11 @@ public override void LoadSubtitle(Subtitle subtitle, List<string> lines, string
             for (int i = 0; i < lines.Count; i++)
             {
                 _lineNumber++;
-                string line = lines[i].TrimEnd();
-                line = line.Trim('\u007F'); // 127=delete acscii
+                string line = lines[i].Replace("\u007F", string.Empty); // 127=delete acscii
+                if (_expecting == ExpectingLine.Number || _expecting == ExpectingLine.TimeCodes)
+                {
+                    line = line.Trim();
```
Comment (ivandrofly) [inline libse/SubtitleFormats/SubRip.cs]: Will only trim if expecting line is number or timecode

File: libse/SubtitleFormats/SubRip.cs [hunk 2]
```diff
@@ -232,7 +235,7 @@ private bool TryReadTimeCodesLine(string line, Paragraph paragraph)
 
             // Removed stuff after timecodes - like subtitle position
             //  - example of position info: 00:02:26,407 --> 00:02:31,356  X1:100 X2:100 Y1:100 Y2:100
-            if (line.Length > 30 && line[29] == ' ')
+            if (line.Length > 30)
```
Comment (ivandrofly) [inline libse/SubtitleFormats/SubRip.cs]: Already trimmed above.

File: libse/SubtitleFormats/SubRip.cs [hunk 3]
```diff
@@ -180,7 +183,7 @@ private void ReadLine(Subtitle subtitle, string line, string next, string nextNe
 
                         if (_paragraph.Text.Length > 0)
                             _paragraph.Text += Environment.NewLine;
-                        _paragraph.Text += RemoveBadChars(line).TrimEnd().Replace(Environment.NewLine + Environment.NewLine, Environment.NewLine);
+                        _paragraph.Text += RemoveBadChars(line).Replace(Environment.NewLine + Environment.NewLine, Environment.NewLine);
```
Comment (ivandrofly) [inline libse/SubtitleFormats/SubRip.cs]: Tadaa 😄!!! I found something: calling **Replace(Environment.NewLine + Environment.NewLine, Environment.NewLine);** is redundant, line is element of "text.SplitToLines()"; right?

I'll send patch later, so it doesn't conflict with this one.

**Comment by niksedk** [comment]
If this is still relevant please add a new pr with info about why, what and unit tests

**Pull request closed without merging**